decidim-accountability/lib/decidim/api/result_type.rb

```diff
@@ -26,6 +26,17 @@ class ResultType < Decidim::Api::Types::BaseObject
       field :parent, Decidim::Accountability::ResultType, "The parent result", null: true
       field :status, Decidim::Accountability::StatusType, "The status for this result", null: true
       field :timeline_entries, [Decidim::Accountability::TimelineEntryType, { null: true }], "The timeline entries for this result", null: true
+
+      def self.authorized?(object, context)
+        context[:debate] = object
+
+        chain = [
+          allowed_to?(:read, :participatory_space, object, context),
+          allowed_to?(:read, :component, object, context)
+        ].all?
```

Comment:
@ahukkanen , @andreslucena  i have doubts regarding the implementation ... I am thinking that this snippet should be : 

```suggestion
        chain = [
          allowed_to?(:read, :participatory_space, object.participatory_space, context),
          allowed_to?(:read, :component, object.component, context)
        ].all?
```

Reply:
False alarm, i had handled this.